src/_data/featured_work.yml

```diff
@@ -0,0 +1,34 @@
+- title: Customer Success Strategy
+  description: |
+    Implemented a new Customer Success program, including setting up a Customer Relationship Management (CRM) platform and assigning Account Managers to work with California Transit Agencies.
+  outcome: |
+    Launched a tiered system to provide 189 California transit agencies with a Customer Success Account Manager after successfully demoing a cohort program. CRM activity increased by 62% in 2023.
+  tags:
+    - Project management
+    - Program development
+
+- title: Operational Data Standard (ODS)
+  description: |
+    Convened the Operational Data Standards (ODS) working group to draft ODS v1, an open standard for describing scheduled transit operations by building on GTFS to include personnel and non-revenue service data.
+  outcome: |
+    Five vendors are capable of reading or writing ODS feeds in their software offerings. This standard is available to transit agencies to improve their scheduled transit operations. ODS v2 is in development.
+  tags:
+    - Data standards development
+
+- title: Data + Donuts
+  image: /assets/our_work/datadonuts.png
+  image_alt_text: An image of a logo containing a donut, where the top half displays a pie chart, and the words “Data plus sign Donuts.”
+  description: In collaboration with government partners, Compiler produced and managed monthly community events, Data + Donuts since 2017—past speakers; LA Metro, California Department of Technology (CDT), SCAG and Caltrans.
+  outcome: Successfully relaunched Data + Donuts in person after the pandemic in June 2023. We have been hosting events ever since and livestreaming them for a larger audience for those who don’t want to attend in person.
+  tags:
+    - Project Management
+
+- title: Benefits
+  image: /assets/our_work/benefits.png
+  image_alt_text: An image showing the home screen of a website called Cal-ITP Benefits, where a hand is holding a credit card using a contactless payment reader.
+  description: Developed Cal-ITP Benefits, a digital tool letting transit riders verify eligibility for a reduced fare and link the discount to their bank card so they automatically get their discount when tapping-to-pay.
+  outcome: Benefits launched at MST (Monterey Salinas Transit, CA) and two additional agencies in 2023. Currently supports Senior and Veteran discounts. Future discounts include ADA and low-income.
```

Comment:
`MST (Monterey Salinas Transit, CA)` is kinda weird too. Why abbreviate first and then spell it out after?

Reply:
also Monterey-Salinas is hyphenated

Reply:
We can add the hyphenate and definitely switch this around unless @o-ram had a reason for adding that way.

Reply:
Andy's comments: https://github.com/compilerla/compiler.la/issues/231#issuecomment-2221775655